source/routing/defining-your-routes.md

```diff
@@ -148,29 +148,29 @@ This router creates three routes:
   <tr>
     <td><code>/</code></td>
     <td><code>index</code></td>
-    <td><code>IndexController</code></td>
-    <td><code>IndexRoute</code></td>
+    <td><code>app/controllers/index</code></td>
+    <td><code>app/routes/index</code></td>
     <td><code>index</code></td>
   </tr>
   <tr>
-    <td>N/A</td>
+    <td>N/A</td>n
     <td><code>posts</code><sup>1</sup></td>
-    <td><code>PostsController</code></td>
-    <td><code>PostsRoute</code></td>
+    <td><code>app/controllers/posts</code></td>
+    <td><code>app/routes/posts</code></td>
     <td><code>posts</code></td>
   </tr>
   <tr>
     <td><code>/posts</code></td>
     <td><code>posts.index</code></code></td>
-    <td><code>PostsController</code><br>↳<code>PostsIndexController</code></td>
-    <td><code>PostsRoute</code><br>↳<code>PostsIndexRoute</code></td>
+    <td><code>app/controllers/posts</code><br>↳<code>app/controllers/posts/index</code></td>
+    <td><code>app/routes/posts</code><br>↳<code>app/routes/posts/index</code></td>
     <td><code>posts</code><br>↳<code>posts/index</code></td>
   </tr>
   <tr>
     <td><code>/posts/new</code></td>
     <td><code>posts.new</code></td>
-    <td><code>PostsController</code><br>↳<code>PostsNewController</code></td>
-    <td><code>PostsRoute</code><br>↳<code>PostsNewRoute</code></td>
+    <td><code>app/controllers/posts</code><br>↳<code>app/controllers/posts/new</code></td>
+    <td><code>app/routes/posts</code><br>↳<code>app/posts/new</code></td>
```

Comment:
For this table (and other similar tables), I used there Ember CLI file path. It seemed like the best choice at the time, but I'm not super convinced. Do you think it makes more sense to use the `controller:post` style instead?